contrib/resources/drives/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(FILES
+
+    y/dos/debug.com
+    y/dos/deltree.com
+    y/dos/xcopy.exe
+    y.conf
+)
```

Comment:
Why not just glob `y/*` too here? We shouldn't put anything in to the `y` folder we don't wanna include.